wrTransport.h

```diff
@@ -0,0 +1,56 @@
+#pragma once
+
+#include <stdint.h>
+#include "wrFilter.h" // filter_lp1_t
+
+typedef enum{ transport_motor_standard
+            , transport_motor_quick
+            , transport_motor_instant
+} transport_motor_speed_t;
+
+// TODO: perhaps some of these shouldn't be floats?
+typedef struct std_speeds{
```

Comment:
this is a weird personal one, but i'd remove the `std_speeds` here as the struct doesn't need to be self-referential.